sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -2606,11 +2599,11 @@
         },
         "endTimeUtc": {
           "format": "date-time",
-          "description": "System generated.  Ignored at creation.\r\nRepresents the time the job stopped processing.",
+          "description": "System generated.  Ignored at creation. Represents the time the job stopped processing.",
```

Comment:
I would leave this part out: System generated.  Ignored at creation
We can mark the fields as readonly instead.